src/Pages/PlayQuiz/Question/Question.tsx

```diff
@@ -0,0 +1,242 @@
+            }
+        }
+        if(((event.target as HTMLElement).innerHTML.includes("Submit"))){
+            submitButtonClicked(totalScore+score)
+        }
+        dispatch({
+            type:"SET_SCORE",
+            payload:totalScore+score
+        })
+        dispatch({
+            type:"LOAD_QUESTION",
+            payload:currentQuiz.questions![currentIndex]
+        })
+        setChecked([])
+        setShowHint(false)
+        setTimer(30)
+    }
+    
+    return(
+        <>
+            <div className={classes["question-container"]}>
+                <div className={classes["index-points-timer"]}>
+                    <p>{currentIndex}/{totalQuestions}</p>
+                    <p>Timer: {timer}</p>
+                    <p>Points: {showHint? points/2 :points}</p>
+                </div>
+                <p className={classes["question"]}>
+                    {question}
+                </p>
+                {
+                        multipleCorrect?(
+                            <>
+                                <FormGroup>
+                                    {options && options.map(({content,id})=>
+                                        <FormControlLabel
+                                            key={id}
+                                            control={<Checkbox 
+                                                name={content} 
+                                                checked={checked.some(value=>id===value)} 
+                                                onClick={()=>optionClicked(id!)}
+                                            />}
+                                            label={content}
+                                        />
+                                    )}
+                                </FormGroup>
+                            </>
+                        ):(
+                            <>
+                                <RadioGroup aria-label="option" name="option">
+                                    {
+                                        options?.map(({id,content})=>(
+                                            <FormControlLabel 
+                                                key={id}
+                                                control={<Radio
+                                                    color="primary"
+                                                    name={content} 
+                                                    checked={checked.some(value=>id===value)}
+                                                    onClick={()=>optionClicked(id!)}
+                                                />} 
+                                                label={content}
+                                            />
+                                        ))
+                                    }
+                                </RadioGroup>
+                            </>
+                        )
+                    }
+                    {hint&&
```

Comment:
You should remove it as hint is not been displayed

Reply:
Questions can have hints if the quiz creator chooses to add them